src/bundler.ts

```diff
@@ -284,6 +291,9 @@ export class Bundler {
       return;
     }
 
+    const doc = dom5.nodeWalkAncestors(htmlImport, (node) => !node.parentNode)!;
+    const body = dom5.query(doc, dom5.predicates.hasTagName('body'))!;
```

Comment:
There was some weird and fiddly code in the beforeHiddenDiv/afterHiddenDiv/inHiddenDiv logic that also found the hiddenDiv in different ways.  It is simpler to just query the doc body for it, which is why we get body here.